Sources/Fullscreen/Explore/ExploreView.swift

```diff
@@ -88,6 +89,11 @@ public final class ExploreView: UIView {
                     cell.gridView.remoteImageViewDataSource = self
                     cell.gridView.configure(withItems: viewModels)
                     return cell
+                case .brazeBanner(let viewModel):
+                    guard let banner = viewModel.banner else { return UICollectionViewCell() }
```

Comment:
can we quickly test what happens if we actually return `UICollectionViewCell` here? in case we didn't set banner. Don't want anything too bad to happen 😛  but assume it just won't be visible, which is fine

Reply:
Maybe have an EmptyCell() or something so that we control the appearance of the fallback? Even if UICollectionViewCell is just transparent there is no guarantee it will always be so.

Reply:
Thats a good idea to add an EmptyCell. Will avoid issues in the future when Apple suddenly decides UICollectionViewCell should have a default backgroundcolor or something.